pkg/policy/address.go

```diff
@@ -28,6 +28,20 @@ func (a *Addressing) ExtractMailbox(address string) (string, error) {
 	if a.Config.MailboxNaming == config.LocalNaming {
 		return local, nil
 	}
+	if a.Config.MailboxNaming == config.DomainNaming {
+		// If no domain is specified,m assume this is being
+		// used for mailbox lookup via the API
```

Comment:
```suggestion
		// used for mailbox lookup via the API.
```

Super minor, but Google has beat into me that comments should be complete sentences ending with a period. :)